README.md

```diff
@@ -204,7 +204,7 @@ Microbundle uses the fields from your `package.json` to figure out where it shou
   "main": "dist/foo.js",            // CommonJS bundle
   "umd:main": "dist/foo.umd.js",    // UMD bundle
   "module": "dist/foo.m.js",        // ES Modules bundle
-  "esmodule": "dist/foo.modern.js", // Modern bundle
+  "exports": "dist/foo.modern.js", // Modern ES Modules bundle
```

Comment:
```suggestion
  "exports": {
    "require": "./dist/foo.js",  // CommonJS bundle
    "default": "./dist/foo.modern.js", // Modern ES Modules bundle
  }
```

Reply:
So main is still required for Node.js < 12.0, and the `exports.require` is there to cover CommonJS support for Node.js 12.0+, did I get this right?